fs/metrics/common/metrics.go

```diff
@@ -73,6 +73,10 @@ const (
 
 	// TODO this metric is not available now. This needs to go down to BlobReader where the actuall http call is issued
 	SynchronousBytesFetched = "synchronous_bytes_fetched"
+
+	// Number of times the snapshotter falls back to use a normal overlay mount instead of mounting the layer as a FUSE mount.
+	// Note that a layer not having a ztoc is NOT classified as an error, even though `fs.Mount` returns an error in that case.
+	OverlayMountFallbackCount = "overlay_mount_fallback_count"
```

Comment:
Not sure about the name. Maybe something like `synchronous_layer_fetch` fits better?

Reply:
`synchronous_layer_fetch_count` applies equally to the "no ztoc" case. 

What we really want to communicate with an operator here is that lazy loading for this layer failed. Our naming here and in logs seems like it's also trying to communicate that this isn't fatal for the container launch, but I wonder if this is downplaying the severity. Operators should be monitoring this and alarming if it spikes because it represents a significant degradation in the experience.

With that in mind, what about something like `lazy_mount_failure_count`? Something still feels slightly off about that to me, but I can't quite place what it is.

Reply:
I'm fine with ``overlay_mount_fallback_count``

Reply:
I don't think I've ever heard the term "lazy mount". ``fuse_mount_faillure_count`` sounds more straightforward to me if we report it as an error metric.

Reply:
Changed to `asynchronous_mount_failure_count`.

Reply:
Can we go with @hanyuel's suggestion instead? I think it's the most clear out of the ideas here.